src/modules/VirtConfig.rb

```diff
@@ -108,10 +108,10 @@ def isTumbleweed
       end
       false
     end
-    def isSLED
-      Builtins.y2milestone("Checking to see if this is SLED ...")
+    def isSLES
+      Builtins.y2milestone("Checking to see if this is SLES ...")
       distro = OSRelease.ReleaseName
-      if distro.include? "SLED"
+      if distro.include? "SLES"
```

Comment:
NP: If it's not SLES, you probably want it to return false. This isn't written explictly here, so it will implicitly return the value of the last executed statement which is this 'if'. This gives the same result, but it would be nicer to have an 'else' branch that explicitly returns false. Just a style issue.

Reply:
well, in general I suggest to improve loggin to have useful content and also if is not needed. So method can look like

```
def isSLES
  distro = OSRelease.ReleaseName
  Builtins.y2milestone("Checking '#{distro}' to see if this is SLES ...")
  distro.include? "SLES"
end
```

this way you always see distro string in log and simplify method,

Reply:
@jreidinger but in your example it won't log if SLES is actually detected. That is very likely important. Just logging that it is checking is nice, but the result is missing here.

Reply:
(Yes, I know that it does write that string to the log, but it's not strictly the same as the function result)